### ISSUE TITLE
Suggestion: Tutorial for software downgrade

### ISSUE THREAD

**Issue by cg8440** [Maassoft/ColorControl#85]
Hi. Can you please provide a tutorial for those who want to downgrade software on their LG TVs? I enabled the option, and see it under expert tab, but what do I do next? Do I just insert the USB with firmware file in to my TV, and click update Software?  Thanks.

**Comment by chros73** [comment]
Yes, but it needs to be the USB version of the firmware (that can be downloaded from LG website) and not the NSU (OTA) version.

**Comment by cg8440** [comment]
> Yes, but it needs to be the USB version of the firmware (that can be downloaded from LG website) and not the NSU (OTA) version.


Thanks!
